integration-test/trace-recorder/src/main/resources/integrationTestData/TraceDebuggerExamples/org.examples.hackathon.SimpleProgramNonFailingTest_test.txt

```diff
@@ -4,9 +4,9 @@ SimpleProgramNonFailingTest.test() at :0
   Integer;@1[1] = 2 at SimpleProgramNonFailingTest.kt:9
   Integer;@1[2] = 3 at SimpleProgramNonFailingTest.kt:9
   CollectionsKt.listOf(Integer;@1): Arrays.ArrayList@1 at SimpleProgramNonFailingTest.kt:9
-  SimpleProgramNonFailingTestKt.sum(Arrays.ArrayList@1): 3 at SimpleProgramNonFailingTest.kt:10
-    numbers.get(0): 1 at SimpleProgramNonFailingTest.kt:22
+  SimpleProgramNonFailingTestKt.sum(numbers): 3 at SimpleProgramNonFailingTest.kt:10
+    numbers.get(i -> 0): 1 at SimpleProgramNonFailingTest.kt:22
```

Comment:
Use a special arrow symbol instead: ➜

Reply:
and decide whether you need a whitespace here

Reply:
> Bonus not using arrow, but colon instead

Colons collide with type annotations:
```
fun assertEquals(expected: String, actual: String)
```

Using `->` is at least consistent with how we display read trace points in the trace.